app/user.go

```diff
@@ -397,7 +397,7 @@ func (a *App) CreateOAuthUser(service string, userData io.Reader, teamId string)
 
 		err = a.AddDirectChannels(teamId, user)
 		if err != nil {
-			mlog.Error(err.Error())
+			mlog.Error("There was a error with adding direct channel", mlog.Err(err))
```

Comment:
For consistency I would like to go with 
```suggestion
			mlog.Error("Failed to add direct channels", mlog.Err(err))
```